traffic_ops/traffic_ops_golang/cdni/shared.go

```diff
@@ -73,14 +72,16 @@ func GetCapabilities(w http.ResponseWriter, r *http.Request) {
 	}
 	defer inf.Close()
 
-	if inf.Config.Cdni == nil || inf.Config.Cdni.JwtDecodingSecret == "" || inf.Config.Cdni.DCdnId == "" {
+	if inf.Config.Cdni == nil || inf.Config.Secrets[0] == "" || inf.Config.Cdni.DCdnId == "" {
```

Comment:
Unless it is being done elsewhere, we should probably check to make sure that `inf.Config.Secrets` has length of `>=1` before getting the first item from there.

Reply:
this is handled at start up in config.go ParseConfig so if Secrets has a length of 0 it will fail at startup